telco-hub/configuration/reference-crs-kube-compare/required/acm/acmAgentServiceConfig.yaml

```diff
@@ -0,0 +1,39 @@
+---
+apiVersion: agent-install.openshift.io/v1beta1
+kind: AgentServiceConfig
+metadata:
+  name: agent
+spec:
+  databaseStorage:
+    storageClassName: {{ .spec.databaseStorage.storageClassName }}
+    accessModes:
+      - ReadWriteOnce
+    resources:
+      requests:
+        storage: 20Gi
```

Comment:
These storage values should check for compliance against a minimum value. (here and below)

Reply:
We don't have any "easy" way to do this, but it may not be too hard to write a helper template to do it.  In the short term, it's better to just "allow any value" if this is something customers will be customizing, and open up a new story &PR to restrict the value to a minimum value.